src/emulator/auth/operations.ts

```diff
@@ -2421,8 +2458,17 @@ function fakeFetchUserInfoFromIdp(
       });
       break;
     }
+    case providerId.match(/^saml\./)?.input:
+      response.email =
+        samlResponse?.assertion?.subject?.nameId &&
+        isValidEmailAddress(samlResponse.assertion.subject.nameId)
+          ? samlResponse.assertion.subject.nameId
+          : response.email;
+      response.emailVerified = true;
+      response.rawUserInfo = JSON.stringify(samlResponse?.assertion?.attributeStatements);
+      break;
```

Comment:
Open question: Shall we also add attributeStatements to `firebase.sign_in_attributes` of the issued token?

Reply:
Ah yes, I think that's pretty reasonable! Added